### PR TITLE
chore (refs T36853): add missing Props.

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://yaits.demos-deutschland.de/T36853
https://demoseurope.youtrack.cloud/issue/DPP-2744/Missing-Hooks-Fur-Stellungnahme-Tabelle

- add missing Props.

### CODE REVIEW

File: src/components/DpDataTable/DpWrapTrigger.vue [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <template>
-  <button class="c-data-table__wrap-trigger o-link--default btn--blank">
+  <button
+      :data-cy="dataCy !== '' ? dataCy : false"
+      class="c-data-table__wrap-trigger o-link--default btn--blank">
```
Comment (salisdemos) [inline src/components/DpDataTable/DpWrapTrigger.vue:4]: alphabetic order please

**Review by salisdemos — CHANGES_REQUESTED** [review]

File: src/components/DpDataTable/DpWrapTrigger.vue (hunk 1 above)
Comment (ahmad-demos) [reply src/components/DpDataTable/DpWrapTrigger.vue]: https://github.com/demos-europe/demosplan-ui/pull/812/commits/5f89f122bc3fb1d618aed85b1f866613115d8ec1

File: src/components/DpDataTable/DpTableRow.vue [hunk 2]
```diff
@@ -79,7 +79,9 @@
       :class="{ 'is-open': expanded }"
       :title="Translator.trans(expanded ? 'aria.collapse' : 'aria.expand')"
       @click="toggleExpand(item[trackBy])">
-      <dp-wrap-trigger :expanded="expanded" />
+      <dp-wrap-trigger
+          :data-cy="`isExpandableWrapTrigger:${$attrs.index}`"
+          :expanded="expanded" />
```
Comment (salisdemos) [inline src/components/DpDataTable/DpTableRow.vue:84]: please indent with 2 blank spaces, not with 4. For all changes....
And please configure Your IDE to do so by default for vue and js files

**Review by salisdemos — CHANGES_REQUESTED** [review]

File: src/components/DpDataTable/DpTableRow.vue (hunk 2 above)
Comment (ahmad-demos) [reply src/components/DpDataTable/DpTableRow.vue]: https://github.com/demos-europe/demosplan-ui/pull/812/commits/529a27ec30bc0ff1f5bd4f5e2c2285c3c74fee4b

File: src/components/DpDataTable/DpTableHeader.vue [hunk 3]
```diff
@@ -50,14 +50,18 @@
       scope="col"
       class="c-data-table__cell--narrow"
       @click="toggleExpandAll()">
-      <dp-wrap-trigger :title="translations.headerExpandHint" />
+      <dp-wrap-trigger
+          data-cy="isExpandableWrapTrigger"
+          :title="translations.headerExpandHint" />
```
Comment (salisdemos) [inline src/components/DpDataTable/DpTableHeader.vue:55]: indentation please

File: src/components/DpDataTable/DpTableHeader.vue [hunk 4]
```diff
@@ -50,14 +50,18 @@
...
     </th>
     <th
       v-if="isTruncatable"
       scope="col"
       class="c-data-table__cell--narrow"
       @click="toggleWrapAll()">
-      <dp-wrap-trigger :title="translations.headerExpandHint" />
+      <dp-wrap-trigger
+          data-cy="isTruncatableWrapTrigger"
+          :title="translations.headerExpandHint" />
```
Comment (salisdemos) [inline src/components/DpDataTable/DpTableHeader.vue:64]: indentation please

**Review by salisdemos — CHANGES_REQUESTED** [review]

File: src/components/DpDataTable/DpTableHeader.vue (hunk 3 above)
Comment (ahmad-demos) [reply src/components/DpDataTable/DpTableHeader.vue]: Sorry
Done in https://github.com/demos-europe/demosplan-ui/pull/812/commits/5a25e0827b2af71c1085201891d8c1a53e6c9c1b

File: src/components/DpDataTable/DpTableHeader.vue (hunk 4 above)
Comment (ahmad-demos) [reply src/components/DpDataTable/DpTableHeader.vue]: Done in https://github.com/demos-europe/demosplan-ui/pull/812/commits/5a25e0827b2af71c1085201891d8c1a53e6c9c1b

**Review by salisdemos — APPROVED** [review]

**Pull request merged**